common/config_schema.go

```diff
@@ -13,17 +13,6 @@ type SchemaObject struct {
 	ElementName        string `json:"element_name,omitempty" msgpack:"element_name,omitempty"`
 	ElementDescription string `json:"element_desc,omitempty" msgpack:"element_desc,omitempty"`
 
-	// legacy fields
-	// -------------------------------------------
```

Comment:
I've checked that we're not using these fields anymore on any extension and that we aren't referencing them anywhere on the front-end app